### ISSUE TITLE
Receiving delayed duplicates of tabs sent from Firefox desktop

### ISSUE THREAD

**Issue by cpeterso** [mozilla-mobile/firefox-ios#13136]
## Steps to reproduce
Three times over the last two days, I sent a tab from my Firefox desktop to my Firefox iOS. 

### Expected behavior
I should receive each sent tab only once. 

### Actual behavior
I received the sent tab, but then some hours later (maybe 8-24?) I receive a duplicate of the sent tab. 

I don’t remember if Firefox iOS was foregrounded or backgrounded when I received the original tabs, but it was definitely foregrounded when I received the duplicates. I remember the duplicate tabs opening while I was trying to read some other tab. 

### Device & build information
* Device: iPhone 12 running iOS 16.2
* Build version: Firefox iOS Testflight version 9000 (27321)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-5699)

**Comment by dnarcese** [comment]
@tarikeshaq any idea of recent send tab change that could've caused this?

**Comment by cpeterso** [comment]
FWIW, I haven't received any more duplicate tabs since I reported this bug about the three duplicates tabs on Feb 3-4.

**Comment by cpeterso** [comment]
@dnarcese This issue should be re-opened because PR https://github.com/mozilla/application-services/pull/5377 has not been merged and I still experience this bug in Firefox iOS 9000 (31007).

**Comment by tarikeshaq** [comment]
Yes this ticket should be reopened, https://github.com/mozilla-mobile/firefox-ios/pull/14305 should fix it for good

**Comment by cpeterso** [comment]
> Yes this ticket should be reopened, https://github.com/mozilla-mobile/firefox-ios/pull/14305 should fix it for good

#14305 was merged in June 2023, so I think we can close this issue now.
